### ISSUE TITLE
'CppcheckPlugin' Description: Could not load file or assembly 'Microsoft.VisualStudio.Threading, Version=17.3.0.0

### ISSUE THREAD

**Issue by jjkalafut** [VioletGiraffe/cppcheck-vs-addin#224]
I'm having this issue after installing in VS2022.
I did notice that app.config has <assemblyIdentity name="Microsoft.VisualStudio.Threading" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
        <bindingRedirect oldVersion="0.0.0.0-15.8.0.0" newVersion="15.8.0.0"/>

Perhaps that needs to be 17.3?

I do have cppcheck installed and working on the PC, just the extension is giving me issues.

**Comment by jjkalafut** [comment]
markup issue in my first commen, should be app.config has: bindingRedirect oldVersion="0.0.0.0-15.8.0.0" newVersion="15.8.0.0"

**Comment by VioletGiraffe** [comment]
Which release have you tried, is it the new 1.5 for VS2022?

**Comment by jjkalafut** [comment]
Yep, i haven't used any older ones. Downloaded today. The error is the normal 'plugin couldnt load', but i coped that reference from the vs activity log.

**Comment by 0x416c69** [comment]
@VioletGiraffe Same problem here. Using Visual Studio Enterprise 2022 - LTSC 17.2.8

**Comment by boolCoder99** [comment]
Had the same issue and could "solve" it. Visual Studio used it's own Threading.dll of version 17.1.xx located at Visual Studio 2022\Common7\IDE\PublicAssemblies\Microsoft.VisualStudio.Threading.17.x 
I just downgraded all the Dependencies in Nuget to 17.1.xx and rebuilt the plugin. Now it works for me.
Upgrading the DLL resulted in VS not starting anymore :/
